### ISSUE TITLE
--float32 parameter no longer recognized in v3

### ISSUE THREAD

**Issue by HENDRIX-ZT2** [facebookresearch/demucs#253]
## 🐛 Bug Report

`--float32` used to work fine in earlier version, but v3 no longer accepts this flag.

## To Reproduce

Add `--float32` to command.

## Expected behavior

A float wav file is saved.

## Actual Behavior
The following error is raised:

`usage: demucs.separate [-h] [-s SIG | -n NAME] [--repo REPO] [-v] [-o OUT]
                       [-d DEVICE] [--shifts SHIFTS] [--overlap OVERLAP]
                       [--no-split] [--mp3] [--mp3-bitrate MP3_BITRATE]
                       tracks [tracks ...]
demucs.separate: error: unrecognized arguments: --float32`

## Your Environment

- Python and PyTorch version: 3.7.8, 1.9.0+cu111
- Operating system and version (desktop or mobile): Win10
- Hardware (gpu or cpu, amount of RAM etc.): RTX3060

**Comment by adefossez** [comment]
Yes, this was removed because I did not see a strong use case for this. why do you need float32 wav files ?

**Comment by HENDRIX-ZT2** [comment]
For further processing, you don't want to add a dithering step inbetween as it adds unneeded distortion and noise. So keeping the data float32 would be preferred.

**Comment by louie-github** [comment]
Also, this option is still mentioned in the README even if the latest version does not support it, and does not list it as a supported argument. I was personally a little confused by that.

**Comment by HENDRIX-ZT2** [comment]
The latest commit has removed the flag from the readme. :(
So is there no chance of getting back float32 output?

**Comment by adefossez** [comment]
I've added back the parameter with this commit 
https://github.com/facebookresearch/demucs/commit/4826b86a1e079d7c7d3c5c7e741911a5ff25c081

For now you will need to install Demucs from the repo (see instructions in the readme), until I release a new version on PyPi.
